spec/jobs/sufia_resque_queue_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'spec_helper'
+
+describe Sufia::Resque::Queue do
+  let (:subject) { Sufia::Resque::Queue.new "test_queue" }
+  let (:job) { double('job') }
+
+  context "no errors" do
+    it "queues the job" do
+      expect(::Resque).to receive(:enqueue_to).and_return(true)
+      subject.push(job)
+    end
+  end
+
+  context "timeout error" do
+    it "retries on timeout" do
+      expect(::Resque).to receive(:enqueue_to).and_raise(Redis::TimeoutError)
```

Comment:
It looks here like you may be conflating stubs with tests here.  I would expect these tests to set the right stubs in before blocks that trigger timeouts and successful runs and then the tests to make sure the behavior at a higher layer is as expected.